docs/documentation/admin/workflows/admin_workflow_groups.md

```diff
@@ -0,0 +1,264 @@
+---
+id: admin_workflow_groups
+title: I. Workflow Groups
+sidebar_label: I. Workflow Groups
+---
+import useBaseUrl from '@docusaurus/useBaseUrl'; 
+import Highlight from '@theme/Highlight';
+
+<img alt="design" className="img_sizing" src={useBaseUrl('img/design/Workflows.svg')} />
+<br/>
+
+<div className="alert alert--secondary">
+
+## Summary
+Learn here how to create and edit _workflow groups_. Familiarize yourself with the [initial settings panel](#initial-settings-panel) and the [create/edit workflow group settings panel](#create-edit-a-workflow-group).
+
+</div>
+<br/>
+
+
+## Initial Settings Panel {#initial-settings-panel}
+From the initial settings panel, you can find the entire list of _workflow groups_ that have been created. Read below to see how you can: 
+- [Create and edit workflow groups](#create-edit-a-workflow-group)
+- [Configure workflows](/docs/documentation/admin/workflows/admin_workflow_configure#workflows-settings-panel)
+- [View workflow group channels](/docs/documentation/admin/admin_channels#edit--create-single-channel)
+
+_Workflow groups settings panel shown below:_
+
+<img alt="workflow group" className="img_sizing item shadow--tl" src={useBaseUrl('img/admin_workflows_groups_00.png')} />
+<br/><br/>
+
+
+**Create a new workflow group:** New _workflow groups_ can be created by pressing the <span class="badge badge--secondary">+</span> icon found in the upper right-hand corner. Once you press the icon, the [create a workflow group settings panel](#create-edit-a-workflow-group) will open up.
+
+
+Further icon descriptions can be found in the [Overview section](/docs/documentation/admin/admin_overview).
+
+_The "Initial Settings Panel" table information is explained below:_
+- **Icon**: Group icon
+- **Name**: Visible group name.
+- **Initial Workflow**: Workflow which starts off the task.
+- **Options**: _Configure Workflows_ - _View Channels_ - _Edit Workflow Group_
+
+:::note
+  _**Configure Workflows**, **View Channels**, and **Edit Workflow Group** options are represented respectively by the following icons:_
+
+<img alt="workflow group" className="img_sizing item shadow--tl" src={useBaseUrl('img/admin_workflows_options_00.png')} />
+<br/>
+
+:::
+
+<br/>
+
+## Create/Edit a Workflow Group {#create-edit-a-workflow-group}
+After choosing to create or edit a _workflow group_ from the [initial settings panel](#initial-settings-panel), you will see the following settings panel:
+
+<img alt="workflow group settings panel" className="img_sizing item shadow--tl" src={useBaseUrl('img/admin_workflows_create_group_00.png')} />
+<br/><br/>
+
+
+#### In the following sub-sections, we will review each section of the settings panel in detail: {#sub-sections}
```

Comment:
All these next steps are shared with "nomal-group" configurations, and some with links.

Reply:
I'll fix this once I re-do the Groups section.